include/tvTypes.h

```diff
@@ -371,6 +371,13 @@ typedef enum tvPQParameterIndex {
     PQ_PARAM_CMS_LUMA_YELLOW,                    //!< Picture parmaeter is Component luma yellow
     PQ_PARAM_CMS_LUMA_CYAN,                     //!< Picture parmaeter is Component luma cyan
     PQ_PARAM_CMS_LUMA_MAGENTA,                  //!< Picture parmaeter is Component luma magenta
+    PQ_PARAM_WB_RED_GAIN,                       //!< Picture parmaeter is Component Red Gain
+    PQ_PARAM_WB_GREEN_GAIN,                     //!< Picture parmaeter is Component Green Gain
+    PQ_PARAM_WB_BLUE_GAIN,                      //!< Picture parmaeter is Component Blue Gain
+    PQ_PARAM_WB_RED_OFFSET,                     //!< Picture parmaeter is Component Red Offset
+    PQ_PARAM_WB_GREEN_OFFSET,                   //!< Picture parmaeter is Component Blue offset
```

Comment:
Are these new parameters expected to be tested only for SaveCustom2PointWhiteBalance?

Reply:
No. This tvPQParameterIndex_t Enum params will be used to test GetDefaultPQParams( ) and GetPQParams( ).
In both the function this Enum will be used to fetch the corresponding values for current source/pqmode/format from driver.